geomstats/geometry/positive_reals.py

```diff
@@ -0,0 +1,318 @@
+    centered Gaussian autoregressive time series machine learning
+    in Poincaré and Siegel disks: application for audio and radar
+    clutter classification, PhD thesis, 2022
+.. [Cabanes_CESAR_2019] Yann Cabanes, Frédéric Barbaresco, Marc Arnaudon,
+    Jérémie Bigot. Unsupervised Machine Learning for Pathological Radar
+    Clutter Clustering: the P-Mean-Shift Algorithm, C&ESAR 2019, Nov 2019,
+    Rennes, France. hal-02875430
+    https://hal.archives-ouvertes.fr/hal-02875430/document
+.. [Cabanes_GSI_2019] Yann Cabanes, Frédéric Barbaresco, Marc Arnaudon,
+    Jérémie Bigot. Toeplitz Hermitian Positive Definite Matrix Machine Learning
+    based on Fisher Metric. Geometric Science of Information, 2019,
+    10.1007/978-3-030-26980-7_27. hal-02875403
+    https://hal.archives-ouvertes.fr/hal-02875403/document
+.. [Cabanes_RADAR_2019] Yann Cabanes, Frédéric Barbaresco, Marc Arnaudon,
+    Jérémie Bigot. Non-Supervised High Resolution Doppler Machine Learning
+    for Pathological Radar Clutter. RADAR 2019, Sep 2020, Toulon, France.
+    10.1109/RADAR41533.2019.171295. hal-02875415
+    https://hal.archives-ouvertes.fr/hal-02875415/document
+.. [JV2016] B. Jeuris and R. Vandebril. The Kahler mean of Block-Toeplitz
+    matrices with Toeplitz structured blocks, 2016.
+    https://epubs.siam.org/doi/pdf/10.1137/15M102112X
+"""
+
+import geomstats.backend as gs
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class PositiveReals(OpenSet):
+    """Class for the manifold of positive reals.
+
+    The real positive axis endowed with the Information geometry metric.
+
+    Parameters
+    ----------
+    scale : float
+        Scale of the positive reals metric.
+        Optional, default: 1.
+    """
+
+    def __init__(self, scale=1.0, **kwargs):
+        super().__init__(
+            dim=1,
+            embedding_space=Euclidean(1),
+            metric=PositiveRealsMetric(scale=scale),
+            **kwargs
+        )
+        self.scale = scale
+
+    @staticmethod
+    def belongs(point, atol=gs.atol):
+        """Check if a point is a positive real.
+
+        Evaluate if a point belongs to the positive real axis,
+        i.e. evaluate if its norm is lower than one.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., 1]
+            Points to be checked.
+        atol : float
+            Tolerance.
+            Optional, default: backend atol.
+
+        Returns
+        -------
+        belongs : array-like, shape=[...,]
+            Boolean denoting if point is a positive real.
+        """
+        is_scalar = gs.ndim(point) == 1 or (gs.ndim(point) == 2 and point.shape[1] == 1)
+        if not is_scalar:
+            return gs.zeros([point.shape[0]], dtype=bool)
+        point = gs.reshape(point, (-1,))
+        is_real = gs.imag(point) == 0
```

Comment:
should we use tolerance instead of equality? e.g. `gs.abs(gs.imag(point)) < gs.atol`